source/index.html.md.erb

```diff
@@ -3019,7 +3056,7 @@ String temporaryUrl = ss.imageUrlResources().getImageUrls(imageUrls).getImageUrl
 			"imageId": "g2jdKdfhQa_abKJmPnhC_B",
 			"error": {
 				"errorCode": 5001,
-				"message": "localized_error_message"
+				"message": "customizable error message"
```

Comment:
"Customizable" implies that the client can change it. These are errors that we return to them. If you can get a concrete example from Daniel, that would be great. Else something like "You do not have permission to view this image."

Reply:
Hmm. Here's what our error code table shows: "400 | 5xxx | Errors in the 5xxx range represent conditions that a developer cannot reasonably prevent or handle, most typically related to account status. These error messages are localized and can be displayed to the end-user to inform them of the condition that caused the error to occur." 
My personal take is that "localized" is being used incorrectly in both places. I think we should fix it in both places once we agree on a term.  "reserved" or the example you give works for the actual sample, and then some other substitution for the table?